modules/virt-creating-a-service-from-a-virtual-machine.adoc

```diff
@@ -0,0 +1,232 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-using-the-default-pod-network-with-virt.adoc
+
+[id="virt-creating-a-service-from-a-virtual-machine_{context}"]
+
+= Creating a service from a virtual machine
+
+Create a service from a running virtual machine by first creating a Service object to expose the virtual machine.
```

Comment:
It looks like you're mostly using the updated API object formatting guidelines throughout, so I would expect something like "Service object" to have been uppercased+backticked:

```
`Service` object
```

Same question for some of the "Service YAML" instances throughout. Seems like those could be uppercase+backtick, or I could possibly see an argument for lowercase+plaintext if that was to be considered a general reference, but uppercase+plaintext seems like it's caught in the middle.

Reply:
Yeah... again, it's a case of being told something else by other reviewers. My thinking has been to just been to uppercase object name as API uses it. But since we ref the YAML it does make sense to tick it. Same for "Service YAML" as you say. 

So I changed all instances of Service to `Service` for Service object and Service YAML.